lib/_stream_readable.js

```diff
@@ -803,9 +803,11 @@ Readable.prototype.wrap = function(stream) {
   // important when wrapping filters and duplexes.
   for (var i in stream) {
     if (this[i] === undefined && typeof stream[i] === 'function') {
-      this[i] = function(method) { return function() {
-        return stream[method].apply(stream, arguments);
-      }; }(i);
+      this[i] = function(method) {
+        return function() {
+          return stream[method].apply(stream, arguments);
+        };
+      }(i);
```

Comment:
Unrelated to this change, but I think one level of function wrapping can be removed here:

```js
this[i] = function(method) {
  return stream[method].apply(stream, arguments);
}(i);
```